.github/auto-labeler.yml

```
@@ -1,23 +1,51 @@
+      body: '#breaking'
+  - label: 'enhancement'
     sync: true
     matcher:
-      title: '^docs:.*'
+      title: '^[fF]eat.*'
   - label: 'bug'
     sync: true
     matcher:
-      title: '^fix:.*'
-  - label: 'ospp-2024'
+      title: '^[fF]ix.*' 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)